libraries/botbuilder-dialogs-declarative/src/defaultLoader.ts

```diff
@@ -39,6 +39,9 @@ export class DefaultLoader implements CustomDeserializer<Configurable, Record<st
                 }
                 instance[`${key}`] = converter.convert(value);
             } else {
+                if (key === 'schema' && typeof value === 'string') {
+                    value = JSON.parse(this._resourceExplorer.getResource(value).readText());
```

Comment:
In order to best preserve existing behavior we should do this: 
- If `key` is `schema` and `value` is string (and possibly if `value.endsWith(.json)`
- Try to load file `value`, try to parse, if success then assign
- Otherwise, assign `value` as before

Reply:
[This work?](https://github.com/microsoft/botbuilder-js/pull/3628/files#diff-7a0e9072626dfb54b4cacb48459f1ce6b0648e86e4894b87b2d5b5b1b6c960e8R42)